chart-values/elasticsearch-curator_values.yaml

```diff
@@ -0,0 +1,88 @@
+# Default values for elasticsearch-curator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cronjob:
+  # At 01:00 every day
+  schedule: "0 9 * * *"
```

Comment:
at 1:00 or 9:00?

Reply:
@angeloskyratzakos we don't really care as this is an examples values file but fixed it thanks ;)